expression/expr_to_pb.go

```diff
@@ -186,6 +186,18 @@ func ToPBFieldType(ft *types.FieldType) *tipb.FieldType {
 	}
 }
 
+// FieldTypeFromPB converts *tipb.FieldType to *types.FieldType.
+func FieldTypeFromPB(ft *tipb.FieldType) *types.FieldType {
```

Comment:
Can we reuse this function in cop_handler_dag.go?

Reply:
em... the paramenter is difference, this is from `*tipb.FieldType`, the other is from `*tipb.ColumnInfo`.

Reply:
A little diiference here, this is decode from FieldTypePB, `cop_handler_dag ` is decode from `columnInfo` PB.